eventstream_rpc/include/aws/eventstreamrpc/EventStreamClient.h

```diff
@@ -487,7 +487,7 @@ namespace Aws
             const ServiceModel &m_serviceModel;
         };
 
-        class AWS_EVENTSTREAMRPC_API ClientOperation : private ClientContinuationHandler
+        class AWS_EVENTSTREAMRPC_API ClientOperation : public ClientContinuationHandler
```

Comment:
trivial: did the inheritance really need to change from private to public?

Reply:
Yes, otherwise the destructor was uncallable.